src/ccstruct/tabletransfer.h

```diff
@@ -0,0 +1,60 @@
+/******************************************************************************
+ * File:        tabletransfer.h
+ * Description: Infrastructure for the transfer of table detection results
+ * Author:      Sintun
+ *
+ * (C) Copyright 2021, Sintun
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ****************************************************************************/
+
+#pragma once
```

Comment:
```suggestion
#ifndef TESSERACT_CCSTRUCT_TABLETRANSFER_H_
#define TESSERACT_CCSTRUCT_TABLETRANSFER_H_
```

Reply:
The current Tesseract code avoids `#pragma once` because it is not a standard (even if many compilers support it).